scripts/unix_share_build.sh

```diff
@@ -25,19 +24,26 @@ function mac_install_prerequisites() {
     brew install eigen gcc@6 gsed
 }
 
-function ubuntu_version() {
-    export UBUNTU_VERSION=$(lsb_release -rs |cut -d"." -f1)
-    echo "Ubuntu Version: " $UBUNTU_VERSION
+function unix_version() {
+    export LINUX_VERSION=$(lsb_release -rs |cut -d"." -f1)
+    echo "Linux Version: " $LINUX_VERSION
 }
 
 function linux_install_prerequisites() {
     TZ=America/Chicago
     $SUDO ln -snf /usr/share/zoneinfo/$TZ /etc/localtime
     $SUDO sh -c 'echo $TZ > /etc/timezone'
     $SUDO apt-get update -y
-    $SUDO apt-get install -y g++ libeigen3-dev patchelf git cmake curl lsb-release python3  lsb-core
+    if [ "$OS" == "ubuntu" ]; then
+    $SUDO apt-get install -y g++ libeigen3-dev patchelf git cmake curl lsb-release python3 lsb-core
```

Comment:
Notice that this script relies on `SUDO` but it never gets set.  Current behavior is probably that this resolves to nothing and doesn't matter... but we should either remove the `$SUDO` here, or set the variable as appropriate.

Reply:
I've now set SUDO to an empty string in the mac and linux github actions scripts in commit 6dc6a66

Reply:
Ah that setting of SUDO to an empty string appears to have not worked